.gitignore

```diff
@@ -14,3 +14,4 @@ node_modules/*
 *.pm.tdy
 *.bs
 .idea/
+logs/standalone_results.log
```

Comment:
Line 10 already ignores `logs/*.log` -- I think that this logfile in particular was committed before the .gitignore was updated... we just need to `git rm` the file.

See drdrew42/renderer#75.

Reply:
Good point.   I missed that.

Reply:
I removed that line.